driver_station/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(driver_station)
+find_package(catkin REQUIRED)
+catkin_package()
+catkin_python_setup()
+install(PROGRAMS scripts/driver_station
```

Comment:
Nitpick: There should be a blank line before the `install` to improve readability as we expand this file. There also should not be an extra blank line at the end of the file - Every file should end in one and exactly one newline